src/pages/Login.jsx

```diff
@@ -0,0 +1,72 @@
+import React, { useRef } from "react";
+
+function Login() {
+  const [isLogged, setIsLogged] = React.useState(false);
+  const [authentificationToken, setAuthentificationToken] = React.useState("");
```

Comment:
authentication, not authentification

and this state would better be in a context :)